dotnet/TF2SA.Data/Services/Mariadb/StatsService.cs

```diff
@@ -94,13 +94,13 @@ select new
 
             var playerStats = (
                 from jps in joinedPlayerStats
-                where jps.ClassId != 7
+                where jps.ClassId != 7 && jps.Playtime > 60
```

Comment:
Can you add a "constants" file for these thresholds and stuff (Like `StatCollectionConstants`)

i.e. `MIN_PLAY_TIME` instead of just `60`. It's more meaningful and readable.
Anywhere where you're arbitrary picking values and thresholds.

Same for the class IDs. instead of
`where jps.ClassId != 7`, `where jps.ClassId != TF2CLASS.MEDIC`